certif/app/controllers/authenticated/sessions/finalize.js

```diff
@@ -69,6 +69,10 @@ export default class SessionsFinalizeController extends Controller {
       await this.session.save({ adapterOptions: { finalization: true } });
       this.showSuccessNotification('Les informations de la session ont été transmises avec succès.');
     } catch (err) {
+      if (err.errors && err.errors[0] && err.errors[0].status === '409') {
+        this.showConfirmModal = false;
+        return this.showErrorNotification(err.errors?.[0]?.detail);
```

Comment:
La plus besoin de verifier vu qu'on est le cas ou c'est passé:
```suggestion
        return this.showErrorNotification(err.errors[0].detail);
```